.gitignore

```diff
@@ -256,6 +256,9 @@
 /misc/stamp-h
 /misc/libtool
 
+# /release/
+/release
```

Comment:
Minor, but I wonder if we should put this under the already existing `distrib` to avoid adding another top level directory?